walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -89,8 +144,21 @@
     }
     // If at the last event and there is still time in the day.
     if (nextTime < END_OF_DAY) {
-      query.add(TimeRange.fromStartEnd(nextTime, END_OF_DAY, /* inclusive= */ true));
+      timeRanges.add(TimeRange.fromStartEnd(nextTime, END_OF_DAY, /* inclusive= */ true));
     }
-    return query;
+    return timeRanges;
+  }
+
+  public Collection<String> combineMandatoryAndOptional(
+      Collection<String> mandatoryAttendees, Collection<String> optionalAttendees) {
+    // Collection of Strings representing all attendees ( Mandatory and Optional )
+    Collection<String> allAttendees = new ArrayList<>();
+    allAttendees.addAll(optionalAttendees);
+    allAttendees.addAll(mandatoryAttendees);
+    return allAttendees;
+  }
+  public List<TimeRange> populateTimeRanges(
+      Collection<com.google.sps.Event> events, Collection<String> attendees, int desiredDuration) {
```

Comment:
Please import this instead of using the fully qualified class name

Reply:
Ack